packages/host/app/components/operator-mode/code-mode.gts

```diff
@@ -530,6 +543,14 @@ export default class CodeMode extends Component<Signature> {
     return undefined;
   }
 
+  private get isSaving() {
+    return (
+      this.waitForSourceCodeWrite.isRunning ||
+      this.saveFileSerializedCard.isRunning ||
+      this.saveCard.isRunning
+    );
+  }
+
```

Comment:
Nice encapsulation 🤩

Reply:
These are the different scenarios that i want to verify a saving indicator appears via percy (since we can't see this via qunit because of the test waiters)

Reply:
i see that one of them isnt working--it's because the save happens so fast with our test filesystem that the save is essentially synchronous. I'll need to add a little extra time into the saving state like I did for the source code saving state